src/content/developers/docs/smart-contracts/index.md

```diff
@@ -92,6 +92,8 @@ Smart contracts alone cannot get information about "real-world" events because t
 
 There are ways to get around this using [oracles](/developers/docs/oracles/).
 
+Smart contracts are also limited to 24kb but, this can be avoided by using [The Diamond Pattern](https://eips.ethereum.org/EIPS/eip-2535).
```

Comment:
```suggestion
Another limitation of smart contracts is the maximum contract size. A smart contract can be a maximum of 24KB or it will run out of gas. This can be circumnavigated by using [The Diamond Pattern](https://eips.ethereum.org/EIPS/eip-2535).
```